chardevice.go

```diff
@@ -170,10 +170,18 @@ func (cdev CharDevice) QemuParams(config *Config) []string {
 	}
 
 	// Legacy serial is special. It does not follow the device + driver model
-	if cdev.Driver != LegacySerial {
+	if cdev.Driver != LegacySerial && cdev.Driver != PCISerial {
 		qemuParams = append(qemuParams, "-device")
 		qemuParams = append(qemuParams, strings.Join(deviceParams, ","))
 	}
+	
+	//appending -serial none and -monitor none to qemuparams if we are using pci-serial
+	if cdev.Driver == PCISerial {
+		qemuParams = append(qemuParams, "-serial")
+		qemuParams = append(qemuParams, "none")
+		qemuParams = append(qemuParams, "-monitor")
+		qemuParams = append(qemuParams, "none")
```

Comment:
This is unrelated to the serial console, so drop this;  we'll continue to configure monitor as we do on x86.